src/main/java/com/amplitude/Options.java

```diff
@@ -0,0 +1,22 @@
+package com.amplitude;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Options {
+
+    /**
+     * Minimum length for user ID or device ID value.
+     */
+    public String minIdLength;
```

Comment:
Would it better to make this an Integer?

Reply:
Sure! fixing.